runtime/arc.js

```diff
@@ -209,7 +209,7 @@ ${this.activeRecipe.toString()}`;
       pecFactory,
       loader,
       storageProviderFactory: manifest._storageProviderFactory,
-      context: manifest
+      context
```

Comment:
Is this intentional? It looks not related to DOM slots.

Reply:
It's necessary for proper round-trip testing. Fwiw, I checked with @shans before I pushed it.